pyproject.toml

```diff
@@ -34,9 +34,9 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "jupyter_client>=6.1.3,<=7.4.1",
+    "jupyter_client>=7.4.4,<9",
```

Comment:
Maybe we can update the title of the PR to also reflect the bump of `jupyter_client`? So it will be visible in the changelog.